src/main/proto/th2_grpc_common/common.proto

```diff
@@ -199,7 +199,10 @@ message MetadataFilter {
   message SimpleFilter {
     FilterOperation operation = 1;
     bool key = 2;
-    string value = 3;
+    oneof king {
```

Comment:
The name for `oneof` block should be chosen as a generic name of all values that are possible in the `oneof` block. I think we can use `filter_value` instead